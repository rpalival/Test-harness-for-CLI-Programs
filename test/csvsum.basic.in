Name,Age,Department,Salary
Alice,30,Engineering,23000
Bob,35,Marketing,55000
Charlie,40,Engineering,80000
Diana,28,Marketing,62000
Eve,25,Design,50000
Frank,38,Design,65000
Grace,32,Engineering,72000