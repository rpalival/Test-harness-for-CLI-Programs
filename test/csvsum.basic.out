Name: {'unique_values': 7, 'most_common': 'Alice', 'frequency_of_most_common': 1, 'all_frequencies': {'Alice': 1, 'Bob': 1, 'Charlie': 1, 'Diana': 1, 'Eve': 1, 'Frank': 1, 'Grace': 1}}
Age: 228.0
Department: {'unique_values': 3, 'most_common': 'Engineering', 'frequency_of_most_common': 3, 'all_frequencies': {'Engineering': 3, 'Marketing': 2, 'Design': 2}}
Salary: 407000.0